### PR TITLE
F#84 rec limit

### PR DESCRIPTION
**Pull request by samsaksfithian**
Closes yamovie/client#84

### CODE REVIEW

File: configs/requests.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ module.exports = {
       providers: null,
       genres: null,
       languages: 'en',
-      release_year_from: 2017,
-      release_year_until: 2018,
+      release_year_from: 2010,
```
Comment (glenflorendo) [inline configs/requests.js]: Change this to `2018`.

File: routes/user.js [hunk 2]
```diff
@@ -3,8 +3,8 @@ const controllers = require('../controllers');
 
 router.post('/signup', controllers.user.signup);
 router.post('/login', controllers.user.login);
-router.post('/watchlist', controllers.user.addMovieToWatchlist);
-router.get('/watchlist/:id', controllers.user.getWatchlistMovies);
-router.delete('/watchlist/:id', controllers.user.deleteWatchlistMovie);
+// router.post('/watchlist', controllers.user.addMovieToWatchlist);
```
Comment (glenflorendo) [inline routes/user.js]: Let's just uncomment this for now.

File: routes/user.js [hunk 3]
```diff
@@ -3,8 +3,8 @@ const controllers = require('../controllers');
...
+// router.get('/watchlist/:id', controllers.user.getWatchlistMovies);
```
Comment (glenflorendo) [inline routes/user.js]: Let's just uncomment this for now.

File: routes/user.js [hunk 4]
```diff
@@ -3,8 +3,8 @@ const controllers = require('../controllers');
...
+// router.delete('/watchlist/:id', controllers.user.deleteWatchlistMovie);
```
Comment (glenflorendo) [inline routes/user.js]: Let's just uncomment this for now.

**Review by glenflorendo — CHANGES_REQUESTED** [review]
Please refer to the comments.

File: configs/requests.js (hunk 1 above)
Comment (samsaksfithian) [reply configs/requests.js]: Fiiiiine 😠

**Review by glenflorendo — APPROVED** [review]

**Pull request merged**